index.js

```diff
@@ -87,7 +95,11 @@ var gs = {
     if (positives.length === 0) throw new Error('Missing positive glob');
 
     // only one positive glob no need to aggregate
-    if (positives.length === 1) return streamFromPositive(positives[0]);
+    if (positives.length === 1) {
+      return streamFromPositive(positives[0]);
+    } else {
+      opt.allowEmpty = true;
```

Comment:
It is true by default, this is just stopping errors from being thrown when there's multiple paths in the array (see the "should not emit error when multiple globs not found" test).

I guess this could be considered an abuse of configuration, but it's certainly the easiest way to do it.